packaging/rpm/kepler.spec

```diff
@@ -11,7 +11,7 @@ URL:            https://github.com/sustainable-computing-io/kepler/
 Source0:        kepler.tar.gz
 
 BuildRequires: systemd
-BuildRequires: clang llvm llvm-devel zlib-devel make libbpf golang
+BuildRequires: clang llvm llvm-devel zlib-devel make libbpf
```

Comment:
isn't `golang` not a  build requirement? 🤔

Reply:
yes, in a plain rpm build we need golang. However our builder image doesn't install golang rpm, we get golang directly from download, and that causes the build process break.

https://github.com/sustainable-computing-io/kepler/actions/runs/7016613096/job/19088182329